jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ManagementUtils.java

```diff
@@ -770,10 +770,6 @@ static void registerAll() {
 				.addInterface(com.ibm.lang.management.JvmCpuMonitorMXBean.class)
 				.validateAndRegister();
 
-			create(OPENJ9_DIAGNOSTICS_MXBEAN_NAME, openj9.lang.management.internal.OpenJ9DiagnosticsMXBeanImpl.getInstance())
-				.addInterface(openj9.lang.management.OpenJ9DiagnosticsMXBean.class)
-				.validateAndRegister();
```

Comment:
Please help me understand why this was removed. This code is specific to Java 8 where there's no possibility that  `OpenJ9DiagnosticsMXBeanImpl.getInstance()` might return `null` (even though the code would properly handle that situation). Why do we no longer want this bean to be registered?

Reply:
Ah good catch, I missed it was specific to 8 (and we didn't test 8). So we need to add it back and add the HotspotBean there too.

Reply:
Opened https://github.com/eclipse-openj9/openj9/pull/17726 to readd the registration calls.